src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -196,6 +196,31 @@ In `config.toml`, add:
 ```toml
 [build]
 target = ["<host_platform>", "aarch64-fuchsia", "x86_64-fuchsia"]
+
+[target.<host_platform>]
+cc = "clang"
+cxx = "clang++"
+linker = "clang++"
+
+[target.x86_64-fuchsia]
+cc = "clang"
+cxx = "clang++"
+linker = "ld.lld"
+
+[target.aarch64-fuchsia]
+cc = "clang"
+cxx = "clang++"
+linker = "ld.lld"
+```
+
+Note that while `clang++` is used to link for the host platform (e.g.
+`x86_64-unknown-linux-gnu`), `lld` is used to link for Fuchsia. You may also
```

Comment:
I think this is already hard coded in rustc for Fuchsia.

Reply:
It's set to use `rust-lld` by default (`compiler/rustc_target/src/spec/fuchsia_base.rs`) so we have to set `lld = true` for that to work. Added that to the build configuration.